miniapp/miniapp_tsgemm.cpp

```diff
@@ -10,14 +10,14 @@
 
 #include <dlaf/blas_tile.h>
 #include <dlaf/communication/datatypes.h>
+#include <dlaf/communication/message.h>
 #include <dlaf/matrix.h>
 
-#include <hpx/dataflow.hpp>
+#include <hpx/async/dataflow.hpp>
+#include <hpx/execution/executors/pool_executor.hpp>  // > HPX v.1.4.1
```

Comment:
Are all the HPX executor changes compatible with 1.4.0 or just from 1.4.1+ (or even higher)?

In case, you may want to update also HPX requirements
https://github.com/eth-cscs/DLA-Future/blob/fc8429c9d2dea03fd1b9c1bf46329339b29e7ed0/CMakeLists.txt#L63